doc/01-About.md

```diff
@@ -55,15 +42,41 @@ They can be stored either in a database or in .ini files.
 See [here](07-Preferences.md#preferences) for more information about a user's preferences
 and how to configure their storage type.
 
-## Documentation <a id="about-documentation"></a>
+## Modules
+
+Modules extend Icinga Web 2 with additional functionality. They allow the integration of
+capabilities into existing views and even other modules. Whether it's a graph provider such
+as [Graphite](https://github.com/Icinga/icingaweb2-module-graphite) or a UI for the Icinga 2
+configuration like the [Director](https://github.com/Icinga/icingaweb2-module-director).
+
+See [here](08-Modules.md#modules) for information on how to install and configure modules.
+
+### The monitoring module <a id="about-monitoring"></a>
+
+> **Note for Icinga DB Users**
+>
+> This module is only for the IDO backend. Use [Icinga DB Web](https://github.com/Icinga/icingadb-web) instead.
+
+This is the core module for most Icinga Web 2 users.
+
+It provides an intuitive user interface for monitoring with Icinga 2.
+There are lots of list and detail views (e.g. for hosts and services)
+you can sort and filter depending on what you want to see.
+
+You can also control the monitoring process itself by sending external commands to Icinga.
+Most such actions (like rescheduling a check) can be done with just a single click.
```

Comment:
It's probably best to hint where the click is (in the UI / the detail view)

Reply:
It's a introduction, not a usage example. I don't think that suits there.

Reply:
I more meant like, that you could just finish the sentence with "can be done with just a single click in the UI." 
Because the way you left it open there puts me under the impression that what follows will answer that implicit mystery and then I was confused when you continued with something else :)

Reply:
Ah, well, added ...`in the UI` to the end :laughing: